shared/parcel.js

```diff
@@ -84,8 +84,11 @@ export function areConnected(parcels, parcelId, sideId) {
   if (parcel.district_id && sameDistrict) {
     return true
   }
-  const sameOwner = parcel.owner === sideParcel.owner
-  return parcel.estate_id && parcel.owner && sameOwner
+  const sameEstate = parcel.estate_id === sideParcel.estate_id
+  if (parcel.estate_id && sameEstate) {
```

Comment:
why not `return parcel.estate_id && parcel.estate_id === sideParcel.estate_id` or
```
const isSameEstate = parcel.estate_id === sideParcel.estate_id
return parcel.estate_id && isSameEstate
```

Reply:
I just copied the style of the lines above: 

```js
const sameDistrict = parcel.district_id === sideParcel.district_id
  if (parcel.district_id && sameDistrict) {
    return true
  }
```

but I can change it